DemoApp/DemoApp/TMButtonPage.xaml.cs

```diff
@@ -1,16 +1,104 @@
+using Trimble.Modus.Components;
+using Trimble.Modus.Components.Enums;
+using Size = Trimble.Modus.Components.Enums.Size;
+
 namespace DemoApp;
 
 public partial class TMButtonPage : ContentPage
 {
-	public TMButtonPage()
+    private string _selectedStyle ="Fill";
+    private string _selectedSize ="Default";
+    private bool isDiabled;
+
+    public bool IsDisabled
+    {
+        get => isDiabled;
+        set
+        {
+            if (isDiabled != value)
+            {
+                isDiabled = value;
+                ValidateRadioButtons();
+            }
+        }
+    }
+
+
+    public TMButtonPage()
 	{
+        
 		InitializeComponent();
-	}
+    }
 
     private void button_Clicked(object sender, EventArgs e)
     {
         Console.WriteLine("Clicked");
```

Comment:
If this method is not used please remove it

Reply:
Removed